modules/solid_mechanics/test/tests/recompute_radial_return/cyclic_kinematic_plasticity_incremental_strain.i

```diff
@@ -0,0 +1,147 @@
+
+# This test assumes a Poissons ratio of zero and applies cyclic
+# displacement loading and unloading conditions on the top boundary
+# in the y-direction while fixing the displacement in the x and z
+# directions; thus, only the normal stress and the normal strains in
+# the y direction are compared in this problem.
+
+# The displacements applied during both the loading and unloading
+# phases are equal in magnitude, resulting in symmetric cyclic loading.
+# The displacement boundary conditions in the x and z directions are
+# fixed to prevent lateral movement.
+
+# This test can be turned into a linear combined hardening problem by
+# assigning a positive value to isotropic hardening constant.
+
+[Mesh]
+  file = 1x1x1cube.e
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y disp_z'
+[]
+
+[AuxVariables]
+  [total_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[Functions]
+  [cyclic_top_disp]
+    type = PiecewiseLinear
+    xy_data = '0 0
+    0.05 0.003125
+    0.1 0.00625
+    0.125 0.003125
+    0.15 0
+    0.175 -0.003125
+    0.2 -0.00625
+    0.225 -0.003125
+    0.25 0
+    0.275 0.003125
+    0.3 0.00625
+    0.325 0.003125
+    0.35 0'
+  []
+[]
+
+[Physics]
+  [SolidMechanics]
+    [QuasiStatic]
+      [all]
+        strain = SMALL
+        incremental = true
+        add_variables = true
+        generate_output = 'stress_yy plastic_strain_xx plastic_strain_yy plastic_strain_zz'
+      []
+    []
+  []
+[]
+
+[AuxKernels]
+  [total_strain_yy]
+    type = RankTwoAux
+    rank_two_tensor = total_strain
+    variable = total_strain_yy
+    index_i = 1
+    index_j = 1
+  []
+[]
+
+[BCs]
+  [y_pull_push_function]
+    type = FunctionDirichletBC
+    variable = disp_y
+    boundary = 5
+    function = cyclic_top_disp
+  []
+  [x_bot]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 4
+    value = 0.0
+  []
+  [y_bot]
+    type = DirichletBC
+    variable = disp_y
+    boundary = 3
+    value = 0.0
+  []
+  [z_bot]
+    type = DirichletBC
+    variable = disp_z
+    boundary = 2
+    value = 0.0
+  []
+[]
+
+[Materials]
+  [elasticity_tensor]
+    type = ComputeIsotropicElasticityTensor
+    youngs_modulus = 2.1e5
+    poissons_ratio = 0.0
+  []
+  [combined_plasticity]
+    type = LinearCombinedHardeningStressUpdate
+    yield_stress = 50.0
+    isotropic_hardening_constant = 0
+    kinematic_hardening_modulus = 1000
+  []
+  [radial_return_stress]
+    type = ComputeMultipleInelasticStress
+    tangent_operator = elastic
+    inelastic_models = 'combined_plasticity'
+  []
+[]
+
+[Executioner]
+  type = Transient
+
+  solve_type = 'PJFNK'
+
+  petsc_options = '-snes_ksp_ew'
+  petsc_options_iname = '-ksp_gmres_restart'
+  petsc_options_value = '101'
+
+  line_search = 'none'
+
+  l_max_its = 50
+  nl_max_its = 50
+  nl_rel_tol = 1e-12
+  nl_abs_tol = 1e-10
+  l_tol = 1e-9
+
+  start_time = 0.0
+  end_time = 0.35
+  dt = 0.00125
+  dtmin = 0.0001
+[]
+
+[Outputs]
+  exodus = true
+  csv = true
```

Comment:
Don't output the exodus files.  Do a csvdiff on the csv output. 
Add a post processor for element max of some stress values, maybe von mises stress, and effectiveplastic strain.